smoothie.js

```diff
@@ -229,7 +229,7 @@
 	// Reject NaN
 	if (isNaN(timestamp) || isNaN(value)){
 		return
-	}  
+	}
```

Comment:
I suppose irrelevant changes like this should go to a separate commit.

Reply:
Allright; these were actually automatically applied by my IDE as per `.editorconfig` ;-)